src/modules/85-cv/pages/health-source/common/SelectHealthSourceServices/components/RiskProfile/RiskProfile.tsx

```diff
@@ -40,12 +41,24 @@ export function RiskProfile(props: RiskProfileProps): JSX.Element {
     isTemplate,
     expressions,
     isConnectorRuntimeOrExpression,
-    fieldNames = {}
+    fieldNames = {},
+    riskProfileResponse
   } = props
-  const { error, loading, data } = metricPackResponse
+  const { error, loading, data } = metricPackResponse || {}
```

Comment:
as we will be having either metricPackResponse or riskProfileResponse 

 const { error, loading, data } = metricPackResponse || {}
```suggestion
  const { error, loading, data } = (metricPackResponse || riskProfileResponse) || {}
```